code/modules/mob/living/carbon/human/species/station/ipc/ipc_subspecies.dm

```diff
@@ -76,6 +76,14 @@
 	allowed_accents = list(ACCENT_CETI, ACCENT_GIBSON, ACCENT_SOL, ACCENT_COC, ACCENT_ERIDANI, ACCENT_ERIDANIDREG, ACCENT_ELYRA, ACCENT_KONYAN, ACCENT_JUPITER, ACCENT_MARTIAN, ACCENT_LUNA,
 							ACCENT_HIMEO, ACCENT_VENUS, ACCENT_VENUSJIN, ACCENT_PHONG, ACCENT_SILVERSUN, ACCENT_TTS, ACCENT_EUROPA, ACCENT_EARTH)
 
+/datum/species/machine/shell/get_species(var/reference, var/mob/living/carbon/human/H, var/records)
+	if(reference)
+		return src
+	// it's illegal for shells in Tau Ceti space to not have tags, so their records would have to be falsified
+	if(records && !H.internal_organs_by_name[BP_IPCTAG])
```

Comment:
This is really icky.

Reply:
yeah it don't feel good, dunno how else it'd be done though. i'm all ears if someone has a suggestion, just ping me on discord